wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -7,9 +7,7 @@ <h1 class="w-sr-only">
         {{ title }}
     </h1>
     {# breadcrumbs #}
-    <div class="w-py-3{% if parent_page.is_root %} w-pl-3{% endif %}">
-        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' is_expanded=parent_page.is_root %}
-    </div>
+    {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' is_expanded=parent_page.is_root classname='sm:w-py-3 lg:w-py-7' %}
```

Comment:
Important so we get the scrollbar positioned correctly, and not obstructing elements within the breadcrumb (actions menu)